src/components/security/security.service.ts

```diff
@@ -146,9 +146,7 @@ export function getCVEControlList(payload): Promise<ResponseType> {
             ...response,
```

Comment:
This service is being used at one place only so can we add basic level of typing and fallbacks to cve as well for the values that we are consuming?